articles/cross-origin-authentication/index.md

```diff
@@ -10,9 +9,9 @@ contentType:
 useCase: 
   - strategize
 ---
-# Cross-Origin Authentication
+# Cross-origin Authentication
 
-For most situations, Auth0 recommends that authentication transactions be handled via [Universal Login](/hosted-pages/login). Doing so offers [the easiest and most secure way to authenticate users](guides/login/universal-vs-embedded). However, it is understood that some situations may require that authentication forms be directly embedded in an application. Cross-origin authentication provides a way to do this securely.
+Auth0 strongly recommends that authentication transactions be handled via [Universal Login](/hosted-pages/login). Doing so offers [the easiest and most secure way to authenticate users](guides/login/universal-vs-embedded). However, some situations may require that authentication forms be directly embedded in an application. Cross-origin authentication provides a way to do this however, it is not recommended or secure.
```

Comment:
I think it's probably no good to say "it's not secure, but we offer it". Probably better to just say it's not recommended and leave it there?